### DISCUSSION TITLE
Rename the excecutame to something different from tsc

### DISCUSSION THREAD

**Discussion by pdenapo** [ASDAlexander77/TypeScriptCompiler#58] [Ideas]
The standard typescript compiler from Microsoft (that transpiles to javascript) is also called tsc.
(https://aka.ms/tsc)

I think that it would be better to rename the executable of your project to something else (like llvm-tsc for instance) since I think that 
when testing it is useful to have both programs installed in your system, and this would avoid some unnecessary possible confusion.

This might also cause trouble in the future when some linux distribution might want to package your compiler, as all the different binaries are usually kept together in the same directory (/usr/bin)

**Comment by ASDAlexander77** [comment]
Thank you for sugesstions, It is awlays dificult to find out right name, especially for me.

**Reply by Sinfolke** [reply]
tsbin (TypeScript binary. Means ts to binary)
tsn (TypeScript Native. The native compiler of ts)
tsnc (TypeScript Native Compiler)

But for me the most suitable would be 'tsbin'.

**Comment by ASDAlexander77** [comment]
how about
``ts-lang``
``ts-compile``

**Comment by ASDAlexander77** [comment]
``tscc``?

**Comment by Sinfolke** [comment]
ts-lang seems has no logic
ts-compiler is good but long to write. However i'd use it instead "tsc" for first
meaning of tscc is not understable but also suitable..

tsbinc
tsbc

cts - compile ts
Cts - C of ts code (ts in C)

tsnp - TypeScript Native Packager. This is from pkg which compiles js code into binary by including node and the target code into same binary and executing it then.
The native packager is like packager that clearly translates ts to binary without node and other additional tools.

tsbp - TypeScript Binary Packager. Similarly to the previous, but referce that the ts is packaged to binary.

tssp - TypeScript Straight Packager.
tssc - TypeScript Straight Compile.

tsfp - TypeScript Forward Packager.
tsfc - TypeScript Forward Compile.

**Comment by ASDAlexander77** [comment]
how about 
``ts-aot``? - typescript "ahead of time" compiler

**Comment by ASDAlexander77** [comment]
or ``tsbuild``?

**Comment by pdenapo** [comment]
I would suggest to keep it short as possible. Something like ltsc ? (for LLVM-based tsc?)

**Comment by ASDAlexander77** [comment]
how about ```tysc```  - ``ty`` - type and ``sc`` - script
